snowflake_rules/snowflake_login_without_mfa.py

```diff
@@ -0,0 +1,7 @@
+MFA_EXCEPTIONS = [
+    'PANTHER_READONLY',
+    'PANTHER_ADMIN'
+]
```

Comment:
Should these be exceptions? Are there any others? Put this here vs in the SQL because it seems like it might be a little easier for users edit?